test/integration/integration_test_fixtures.py

```diff
@@ -378,6 +401,19 @@ def http_test_server_fixture(request, server_config):
   f.tearDown()
 
 
+@pytest.fixture(params=determineIpVersionsFromEnvironment())
+def http_test_server_fixture_v2_api(request, server_config):
```

Comment:
Can we include the word "envoy" here also? And maybe "deprecated" too? E.g. `http_test_server_fixture_envoy_deprecated_v2_api`.